bumble/transport/file.py

```diff
@@ -28,7 +30,31 @@
 
 
 # -----------------------------------------------------------------------------
-async def open_file_transport(spec):
+class FileTransport(Transport):
```

Comment:
It is simpler and more encapsulated to have `FileTransport` as a local class in `open_file_transport`. That class isn't really meant to be used by itself, it is just a local helper to tie the source and sink together into a `Transport` object. `open_file_transport` should be typed to return just a `Transport`, without having to reveal its internal details.
The same applies to all the other `open_xxx_transport` functions.

Reply:
Done